Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -760,6 +760,11 @@ extension BrowserViewController: BrowserDelegate {
         let contextMenuHelper = ContextMenuHelper(browser: browser)
         contextMenuHelper.delegate = self
         browser.addHelper(contextMenuHelper, name: ContextMenuHelper.name())
+        
+        //new helper to deal with fragment identifiers / hashed URLs
```

Comment:
I'd remove this comment. It would be useful to have a brief description of what HashchangeHelper is, but that should be added as class documentation in `HashchangeHelper.swift` instead of here, so it's tied to the implementation.